modules/puppet_proxy/dependency_injection/dependencies.rb

```diff
@@ -1,45 +1,46 @@
-require 'puppet_proxy/runtime_configuration'
-
+# rubocop:disable Style/Lambda
```

Comment:
I can disable this cop globally. Can't switch to stabby lambdas yet (which this cop insists on), because 1.8.7.